src/components/aktivitetskrav/vurdering/VurderAktivitetskravModal.tsx

```diff
@@ -12,12 +12,12 @@ const texts = {
   modalContentLabel: "Vurder aktivitetskrav",
 };
 
-export type ModalType = `${
-  | AktivitetskravStatus.AVVENT
-  | AktivitetskravStatus.UNNTAK
-  | AktivitetskravStatus.OPPFYLT
-  | AktivitetskravStatus.IKKE_AKTUELL
-  | AktivitetskravStatus.IKKE_OPPFYLT}`;
+export type ModalType = `${Exclude<
```

Comment:
Endret for at Typescript skal klage om man legger til en ny status (samme gjelder switch/case-endringene i [historikk](https://github.com/navikt/syfomodiaperson/pull/911/files#diff-dad317736cc9360f0e4654471e2343683128b900c7c2fdca237806650cfa8ffeR76) og [alert](https://github.com/navikt/syfomodiaperson/pull/911/files#diff-984bdccaa0cafb820cef80285ef0e168a0feafc6b25cfcb9eb9e86b02d5c7bccR72))